SeniorInterview/StocksAnalyzed/src/Main.java

```diff
@@ -22,5 +23,14 @@ public static void main(String[] args) {
             if (count > 5) break;
             count++;
         }
+
+        System.out.println("\nTOP 3 past variation");
+        System.out.println("==============================");
+
+        // Variacao de alguns dias passados
+        AggregateRequestor ar = new AggregateRequestor();
+        for (int i=0; i<3; i++) {
```

Comment:
poderíamos também usar o próprio tickerList pra iterar nos elementos. já que se trata de uma lista
`tickerList.forEach(tick -> ar.requestTickerDays(tick.getName()), ...)`
```suggestion
        for (int i = 0; i < 3; i++) {
```